ocaml/idl/datamodel.ml

```diff
@@ -5587,6 +5590,33 @@ let storage_plugin =
      ])
     ()
 
+let lvhd_enable_thin_provisioning = call
+   ~name:"enable_thin_provisioning"
+   ~in_oss_since:None
+   ~in_product_since:rel_dundee
+   ~allowed_roles:_R_POOL_ADMIN
+   ~params:[
+     Ref _sr, "SR", "The LVHD SR to upgrade to being thin-provisioned.";
+     Int, "initial_allocation", "The initial amount of space to allocate to a newly-created VDI in bytes";
```

Comment:
It is not about "credits". Whatever needs to be filled by the user, given in this case we are talking about MBs or GBs, cannot be passed in bytes.
If this is an internal thing that the user cannot see, than it is fine, we can pass bytes ourselves.

What I said is that the sm-config fields related to quantum allocation and initial allocation need to be in a human-writable format: I cannot picture anyone writing down 1GB in bytes.

If the default is bytes for this call but this call accepts suffixes like MiB and GiB, than it is fine
for me, as long as we are on the same page.